src/interfaces/rating/stars.vue

```diff
@@ -127,9 +129,9 @@ export default {
   }
 }
 .rating-empty {
-  color: var(--lighter-gray);
+  color: var(--lightest-gray);
   &:after {
-    content: "star_border";
+    content: "star"; // star_border
```

Comment:
Lets get rid of this comment. We have git to maintain the history

```suggestion
    content: "star";
```